caffe2/operators/layer_norm_op.cc

```diff
@@ -174,10 +191,16 @@ to the end.)
         "epsilon",
         "(float) default to 0.001. Small value to be added to the stdev when"
         " dividing out by that value. This prevents division by zero.")
+    .Arg(
+        "elementwise_affine",
```

Comment:
This seems redundant, since we can determine whether we should do the elementwise_affine based on the input size.

Reply:
I think being explicit is always better and more readable. Relying on input size can be confusing. But we should probably add asserts to make sure we're in one of the valid states.

Reply:
It will help to determine whether to compute gradients for gamma and beta.